python/lsst/sims/utils/CompoundCoordinateTransformations.py

```diff
@@ -0,0 +1,493 @@
+
+        return raRad[0], decRad[0]
+
+
+    raRad, decRad = _icrsFromObserved(raObs, decObs,
+                                      obs_metadata=obs, epoch=2000.0,
+                                      includeRefraction=True)
+
+    return raRad, decRad
+
+
+def getRotSkyPos(ra, dec, obs, rotTel):
+    """
+    @param [in] ra is the RA in degrees.  Can be a numpy array or a single value.
+    (In the International Celestial Reference System)
+
+    @param [in] dec is Dec in degrees.  Can be a numpy array or a single value.
+    (In the International Celestial Reference System)
+
+    @param [in] obs is an ObservationMetaData characterizing the telescope pointing
+    and site.
+
+    @param [in] rotTel is rotTelPos in degrees
+    (the angle of the camera rotator).  Can be a numpy array or a single value.
+    If a numpy array, should have the same length as ra and dec.  In this case,
+    each rotTel will be associated with the corresponding ra, dec pair.
+
+    @param [out] rotSkyPos in degrees
+
+    WARNING: As of 13 April 2015, this method does not agree with OpSim on
+    the relationship between rotSkyPos and rotTelPos.  This is due to a
+    discrepancy between the time that OpSim uses as the MJD when calculating
+    rotTelPos and the time that OpSim reports as being the actual expmjd
+    of the exposure (rotTelPos is calculated at the beginning of the exposure;
+    expmjd is reckoned at the middle of the exposure).
+    """
+
+    rotSky = _getRotSkyPos(np.radians(ra), np.radians(dec),
+                           obs, np.radians(rotTel))
+
+    return np.degrees(rotSky)
+
+
+
+def _getRotSkyPos(raRad, decRad, obs, rotTelRad):
+    """
+    @param [in] raRad is the RA in radians.  Can be a numpy array or a single value.
+    (In the International Celestial Reference System)
+
+    @param [in] decRad is Dec in radians.  Can be a numpy array or a single value.
+    (In the International Celestial Reference System)
+
+    @param [in] obs is an ObservationMetaData characterizing the telescope pointing
+    and site.
+
+    @param [in] rotTelRad is rotTelPos in radians
+    (the angle of the camera rotator).  Can be a numpy array or a single value.
+    If a numpy array, should have the same length as raRad and decRad.  In this case,
+    each rotTelRad will be associated with the corresponding raRad, decRad pair.
+
+    @param [out] rotSkyPos in radians
+
+    WARNING: As of 13 April 2015, this method does not agree with OpSim on
+    the relationship between rotSkyPos and rotTelPos.  This is due to a
+    discrepancy between the time that OpSim uses as the MJD when calculating
+    rotTelPos and the time that OpSim reports as being the actual expmjd
+    of the exposure (rotTelPos is calculated at the beginning of the exposure;
+    expmjd is reckoned at the middle of the exposure).
+    """
+    altRad, azRad, paRad = _altAzPaFromRaDec(raRad, decRad, obs)
+
+    #20 March 2015
```

Comment:
Technically, yes it is true in that I don't know where the relationship comes from, but we have checked it against OpSim (as mentioned in the docstring).  I will go ahead and remove the comment.